cmd/client/cmd/quota.go

```diff
@@ -100,16 +109,21 @@ func quota(rootArgs *rootArgs, printf, fatalf shared.FormatFn, name string, amou
 			fatalf("Failed to receive a response from Quota RPC: %v", err)
 		}
 
-		printf("Quota RPC returned %s, amount %v, expiration %v",
-			decodeStatus(response.Result),
-			response.Amount,
-			response.Expiration)
-		dumpAttributes(printf, fatalf, response.AttributeUpdate)
+		<-sem
+
+		if rootArgs.repeat < 10 {
+			printf("Quota RPC returned %s, amount %v, expiration %v",
+				decodeStatus(response.Result),
+				response.Amount,
+				response.Expiration)
+			dumpAttributes(printf, fatalf, response.AttributeUpdate)
+		}
 	}
 
 	if err = stream.CloseSend(); err != nil {
 		fatalf("Failed to close gRPC stream: %v", err)
 	}
+	println("Done in ", time.Since(t), " avg=", (time.Since(t).Nanoseconds()/1000)/int64(rootArgs.repeat))
```

Comment:
let's add unit to the output here (as above) and use `time.Microsecond` or `time.Millisecond` instead of 1000.